x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -55,10 +55,16 @@ export const HostIsolationExceptionsList = () => {
 
   const [itemToDelete, setItemToDelete] = useState<ExceptionListItemSchema | null>(null);
 
+  const includedPoliciesParam = location.included_policies;
+
   const { isLoading, data, error, refetch } = useFetchHostIsolationExceptionsList({
     filter: location.filter,
     page: location.page_index,
     perPage: location.page_size,
+    policies:
+      includedPoliciesParam && includedPoliciesParam !== ''
+        ? includedPoliciesParam.split(',')
```

Comment:
backwards compatibility, it should had been an array of strings from the start but those utils were made to use a comma-separated string.

Reply:
I'm not really sure about it, url params is a comma separated values and the `parsePoliciesToKQL` expects a comma separated values. We don't do this split + join + split again for Trusted Apps, I'm missing something I guess

Reply:
@dasansol92 the helpers I created for these policies use array of strings but the old helpers use comma separated strings.

Reply:
Yeah, this is what I mean, I think the `parsePoliciesAndFilterToKql` should take a string and let the `parsePoliciesToKQL` make the string parsing. In that way we should avoid the split above and the join inside the `parsePoliciesAndFilterToKql`

Reply:
```
 const { isLoading, data, error, refetch } = useFetchHostIsolationExceptionsList({
    filter: location.filter,
    page: location.page_index,
    perPage: location.page_size,
    policies:
      includedPoliciesParam && includedPoliciesParam !== ''
        ? includedPoliciesParam : ...

---------------

export const parsePoliciesAndFilterToKql = ({
  policies,
  kuery,
}: {
  policies?: string;
  kuery?: string;
}): string | undefined => {
  if (!policies || !policies.length) {
    return kuery;
  }

  const policiesKQL = parsePoliciesToKQL(policies, '');
  return `(${policiesKQL})${kuery ? ` AND (${kuery})` : ''}`;
};
```

Reply:
We should not need to pass the problems of url-serialization to our internal helpers. Policies are an array of strings and should had been handled like that from the start. I didn't want to refactor the existing helpers but I also didn't want to keep on passing serialization problems to helpers.

doing a `.join` is not a heavy load for the script and it doesn't really make it more complex in anyway.

Reply:
Spoke offline with Esteban as since we are going to use react-query and we are going to use policies not only from the url (even forced when in Policy details tab) it makes more sense to use arrays inside the `parsePoliciesToKQL`. We are going to fix this in another issue in order to use arrays instead of comma separated strings